src/sentry/models/savedsearch.py

```diff
@@ -18,6 +37,9 @@ class SavedSearch(Model):
     type = models.PositiveSmallIntegerField(default=SearchType.ISSUE.value, null=True)
     name = models.CharField(max_length=128)
     query = models.TextField()
+    sort = models.CharField(
```

Comment:
Let's use `sentry.db.models.fields.text.CharField` here so that we're backed by a `text` column, just makes life easier if we want to change the `max_length`